controllers/volumereplicationgroup_controller.go

```diff
@@ -1898,37 +1979,37 @@ func isVRConditionMet(volRep *volrep.VolumeReplication,
 	return true, ""
 }
 
-func (v *VRGInstance) updateProtectedPVCConditionHelper(name, reason, message, defaultMessage string) {
+func (v *VRGInstance) updatePVCDataReadyConditionHelper(name, reason, message, defaultMessage string) {
 	if message != "" {
-		v.updateProtectedPVCDataCondition(name, reason, message)
+		v.updatePVCDataReadyCondition(name, reason, message)
 
 		return
 	}
 
-	v.updateProtectedPVCDataCondition(name, reason, defaultMessage)
+	v.updatePVCDataReadyCondition(name, reason, defaultMessage)
 }
 
-func (v *VRGInstance) updateProtectedPVCDataCondition(name, reason, message string) {
+func (v *VRGInstance) updatePVCDataReadyCondition(name, reason, message string) {
```

Comment:
@raghavendrabhat The `protected` keyword in ClusterDataProtected condition is the reason why I removed the prefix of `Protected` in this method to not have two 'Protected' in the name of `updatePVCClusterDataProtectedCondition`,  as in to avoid naming it `updateProtectedPVCClusterDataProtectedCondition`.